src/Extensions/GridFieldDetailFormItemRequestExtension.php

```diff
@@ -22,4 +26,27 @@ public function updateBreadcrumbs($crumbs)
             ));
         }
     }
+
+    public function updateItemEditForm($form)
+    {
+        $fields = $form->Fields();
+        if ($this->owner->record instanceof CMSPreviewable &&
+            !$fields->fieldByName('SilverStripeNavigator')
+        ) {
+            $template = Controller::curr()
+                ->getTemplatesWithSuffix('_SilverStripeNavigator');
+            $navigator = SilverStripeNavigator::create($this->owner->record);
+            $field = LiteralField::create(
+                'SilverStripeNavigator',
+                $navigator->renderWith($template)
+            )->setAllowHTML(true);
+            $fields->push($field);
+            $form->addExtraClass('cms-previewable')
+                ->removeExtraClass('cms-panel-padded center');
+            Requirements::javascript(
+                'DNADesign/Elemental:javascript/Elemental.Preview.js'
```

Comment:
the vendor/module syntax will need to match composer.json - so it's probably `dnadesign/silverstripe-elemental:...`

Reply:
Hmm, dead code actually, shifted from a CMS JS approach to the `MetaTags` extension in `ElementalPageExtension`. So I'll just remove it.